### ISSUE TITLE
[E-mails] [Subscriber welcome] Apply the most up-to-date design to the welcome email

### ISSUE THREAD

**Issue by mashikag** [Automattic/wp-calypso#61695]
### Details

`Welcome` email is the one that new user receives after their account was created but only after they confirmed their email address. 

The currently implemented new design looks as following.
![Screenshot 2022-03-10 at 14 31 32](https://user-images.githubusercontent.com/2019970/157860490-74959edc-7151-4293-9620-17d9f4e8ba22.png)

### Checklist

- [x] Update the contents of the email to match the contents of the new design. Use the above screenshot as reference.
- [x] Extend `wpcom_use_2021_email_templates` filter to send the subscription emails with the new design applied to non-wpcom users. See [the patch](D76552-code) for more details.

**Note**: do not implement displaying site icon and using site theme color as part of this task. There is another task that will take care of this.

### Related

Done @ D76552-code

Here is P2 discussion related to the scope of this task: pdgv5Y-lA-p2#comment-150.

**Comment by mashikag** [comment]
Done @ D76552-code

![welcome-email-before-after-desktop](https://user-images.githubusercontent.com/2019970/159299681-eebe897a-bd8c-455d-ad03-057265c41af0.png)

![welcome-email-before-after-mobile](https://user-images.githubusercontent.com/2019970/159299691-a454adfb-a192-4258-ade5-bdd2be152cd5.png)
